app/assets/stylesheets/application.css

```diff
@@ -11,6 +11,7 @@
  * It is generally better to create a new file per style scope.
  *
  *= require decidim
+ *= require finder
```

Comment:
We don't want the code in the file to be available to all pages. We're already loading it in the decidim_finder page via tag